lib/providers/create request page providers/leave_type_provider.dart

```diff
@@ -0,0 +1,6 @@
+import 'package:cobe_task/widgets/create%20request%20widgets/type_of_leave_list.dart';
+import 'package:flutter_riverpod/flutter_riverpod.dart';
+
+final leaveTypeNotifierProvider = StateProvider.autoDispose<String>((ref) {
+  return leaveTypes[0];
+});
```

Comment:
you can use arrow funcition here. mark rename `ref` param as `_`. If a parameter is unused it is marked `_` by convention